### ISSUE TITLE
Stop/Keep Words Case Sensitivity Issue

### ISSUE THREAD

**Issue by JordanHelpsAlot** [WheatonCS/Lexos#346]
Please give us some of the following information, 
That will be really helpful to track the bug:

* **if you are proposing an enhancement, please delete all of these**

@evanlaferriere made a good catch about case sensitivity in stop/keep words. When the user chooses make lowercase they may assume that case no longer matters and provide capitalized words as input for stop/keep words. I'm changing it so that stop/keep words are no longer case-sensitive when used on a text with entirely lower case.

**Comment by JordanHelpsAlot** [comment]
Update on this: I implemented a version of this enhancement but the new version created many more issues than it solved. To analyze the entire text for a lowercase check was too resource demanding for how small an enhancement it was. I removed my addition but kept the feature which removes case-sensitivity if the user actually has "make lowercase" checked which will actually catch most of the user mistakes the original fix would have.
